### ISSUE TITLE
Installing from github gives 404 assets

### ISSUE THREAD

**Issue by hss-mateus** [rswag/rswag#467]
As pointed in #465, the latest release of rswag on RubyGems doesn't work with Rails 7, so i pointed this github repo at my gemfile.

The problem is that rswag-ui assets are not installed when using this workaround. This requires me to manually install them at the gem folder, like the example:

```sh
ASSETS_PATH=$(bin/rails r "print Rswag::Ui.config.assets_root")

yarn --cwd $ASSETS_PATH install
```

A possible solution for this is to release a new version to RubyGems, including the installed assets.

**Comment by agildav** [comment]
👀

**Comment by matiassalles99** [comment]
Yes a new release would be great, I'm trying to upgrade to Rails 7 also

**Comment by anmarchenko** [comment]
Same here, would really appreciate new release. Thanks for your work!

**Comment by strifek** [comment]
The workaround could be to download `swagger-ui-bundle.js`, `swagger-ui-standalone-preset.js` and `swagger-ui.css` from `https://registry.npmjs.org/swagger-ui-dist/-/swagger-ui-dist-3.52.5.tgz` and manually place it in your `public` folder (must match your swagger path, eg. `public/api-docs`) to serve them as a static files.

**Comment by coins-black** [comment]
Has the same bug

**Comment by mxdavis** [comment]
👀

**Comment by joelbiffin** [comment]
👀

**Comment by jlestavel** [comment]
For those who would be looking for a temporary solution:

The Github version of the `rswag-ui` gem does not have the `node_modules` folder, so indeed the `Rswag::Ui.config.assets_root` folder does not exist...

https://github.com/rswag/rswag/blob/a813b63bcc64e9ea49f6416ab4fe023b7f090ad8/rswag-ui/lib/rswag/ui/configuration.rb#L22

The reason of that is that the `node_modules` folder is generated on build (with `npm install`) :

https://github.com/rswag/rswag/blob/a813b63bcc64e9ea49f6416ab4fe023b7f090ad8/ci/build.sh#L20-L21

and is kept with the gem, as you can see in commit 334671cb95da8b665e5b3397f995c1326bc607d1. So we have to wait for a new gem release to have the `node_modules` present in the gem.

In the meantime, what you can do is use a local version of those assets. For our app, as we use webpacker, we added `swagger-ui-dist` to our `package.json`. Then, we ask the gem to use our local `node_modules/swagger-ui-dist` instead of the one that is supposed to be embedded in the gem. The config to do that is:

```ruby
Rswag::Ui.configure do |c|
  c.instance_eval do
    @assets_root = Rails.root.join('node_modules/swagger-ui-dist')
  end
end
```
It's not possible to write `c.assets_root = ...` instead of `c.instance_eval { @assets_root = ... }` because `assets_root` is defined as readonly in the config:

https://github.com/rswag/rswag/blob/a813b63bcc64e9ea49f6416ab4fe023b7f090ad8/rswag-ui/lib/rswag/ui/configuration.rb#L11

By the way, for gem maintainers, I guess it would be good to define it as `attr_accessor` so that people could use a custom version of the JS package...

Hope it can help!

**Comment by BookOfGreg** [comment]
Latest version is shipped. Let us know if there's an issue.

**Comment by hss-mateus** [comment]
Yeah, the problem persists, similar as described in #489. Maybe the production build isn't shipping with all the required files:
![image](https://user-images.githubusercontent.com/58234608/153468188-820e0a3b-d9d9-4974-8d8b-bf4a5efaf816.png)

**Comment by BookOfGreg** [comment]
Ok, cause is likely to be we moved from Travis to GitHub Actions and the deploy script did not re-implement the flow exactly. This is an error in the deploy process and I'm working on this for a 2.5.1 release.

**Comment by BookOfGreg** [comment]
I have now released 2.5.1, Thanks for informing us though sorry I hadn't understood the issue I was about to cause before I caused it.

Please let me know if this issue continues.

**Comment by hss-mateus** [comment]
It's working perfectly now, thanks for the quick response!
